teams/Aragon One/2019.md

```diff
@@ -57,9 +56,7 @@ The aim of this initiative is to increase the richness and flexibility of the ex
 
 Focus areas for this initiative:
 
-- **Stateful permissions**: Allow users to set and manage time-based and use-based permissions.
-
-- **UI for conditional permissions**: the ACL in aragonOS 4 supports conditional permissions through permission parametrization, the Permissions app should display and allow to edit parameters.
+- 🔜 **UI for conditional permissions**: the ACL in aragonOS 4 supports conditional permissions through permission parametrization, the Permissions app should display them.
```

Comment:
Because of the previous comment, we will just allow users to see custom parameters, and not modify them.